api/src/main/java/com/redhat/ipaas/api/v1/rest/DataManager.java

```diff
@@ -146,10 +146,10 @@ public String generatePK(Cache<String, WithId> entityMap) {
 
     public Class<? extends WithId> getClass(String kind) {
         switch (kind.toLowerCase()) {
-            case "component":
-                return Component.class;
-            case "componentgroup":
-                return ComponentGroup.class;
+            case "connector":
```

Comment:
If `KIND` is already defined a `Connector.KIND`, it should be used here, too, as constant.